frontend/src/helpers/dataOverlay.js

```diff
@@ -3,17 +3,20 @@ const DEFAULT_METABOLITE_COLOR = '#9df';
 const DEFAULT_REACTION_COLOR = '#fff';
 
 const DATA_TYPES_COMPONENTS = {
-  transcriptomics: {
+  gene: {
+    description: 'gene (e.g., transcriptomics, proteomics, etc.)',
```

Comment:
If we follow British English, `e.g.` is usually not followed by comma, i.e. 

`e.g. transcriptomics, proteomics, etc.)`